roles/edpm_telemetry/tasks/enable_logging.yml

```diff
@@ -20,13 +20,30 @@
   become: true
   failed_when: false
   register: rsyslog_service_state
+  tags:
+    - edpm_logging
 
 - name: Configure rsyslog if present
   become: true
   when:
     - (rsyslog_service_state is success) and
       ((rsyslog_service_state.status['SubState'] | lower) == 'running')
+  tags:
+    - edpm_logging
   block:
+    - name: Install openssl module support for rsyslog
+      ansible.builtin.dnf:
+        name: rsyslog-openssl
+        state: present
+
+    - name: Copy Openshift CA to the node
+      become: true
+      ansible.builtin.copy:
+        src: "{{ edpm_telemetry_config_src }}/ca-openshift.crt"
+        dest: "/etc/ssl/certs/ca-openshift.crt"
```

Comment:
can we define a var with this path so we can use it here and on molecule as well?

Reply:
I dont really want the user to be able to change this path, as there are other part of the configuration generated by the telemetry-operator which needs to use the same path: https://github.com/openstack-k8s-operators/telemetry-operator/pull/251/files#diff-b9f6028da3c23a90b425ff5d97240a112f428590c3ba0339dee2de54e027912fR11

I will use variables for molecule tests for all the rest of the paths.

Reply:
we were disucssing about having a singel role for doing management of the tls ca and certs in https://github.com/openstack-k8s-operators/edpm-ansible/pull/506

looking at the other tls example  (nova)
https://github.com/openstack-k8s-operators/edpm-ansible/pull/491/files

we are installing the certs to 

/var/lib/openstack/certs/<service name>/ and binding that to /etc/pki/<servicename>/... in the container i.e.

/var/lib/openstack/certs/nova/tls.crt:/etc/pki/nova/server-cert.pem:ro


are you sure /etc/ssl/certs/ca-openshift.crt is the correct path?

would it not be /etc/pki/rsyslog/...


@vakwetu any insight here?

Reply:
Well, according to where the certs should be deployed, it could be anywhere. I chose /etc/ssl/certs for being the default cert path. /etc/pki/rsyslog is as good as the other one, no problem.

But before doing the changes, I am interested in that single role for doing management of the cacerts. The one I need here is the default openshift CA, which signs all the openshift certs. Is nova already getting that cacert from openshift and storing it somewhere (/var/lib/openstack/certs/nova/...)?

Reply:
I changed the path for the cacert to /etc/pki/rsyslog